modules/virt-listing-vmis-web.adoc

```diff
@@ -0,0 +1,21 @@
+
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-manage-vmis-web.adoc
+
+[id="virt-listing-vmis-web_{context}"]
+= Listing standalone virtual machine instances using the web console
+
+Using the web console, you can list and view standalone virtual machine instances (VMIs) in your cluster that are not owned by virtual machines (VMs).
+
+[NOTE]
+====
+When you list VMIs using the CLI, all VMIs including standalone VMIs and VMIs that are owned by VMs and other objects display. The web console displays only standalone VMIs.
+====
+
+
+.Procedure
+
+. Click *Workloads > Virtual Machines* from the side menu. A list of VMs and standalone VMIs display. You can identify standalone VMIs by the dark colored badges that display next to the virtual machine instance names.
+
+. For details, click on a  standalone VMI listed in the  *Name* column. The *virtual machine instance details* page displays.
```

Comment:
`virtual machine instance details` should be `virtual machine instance overview`.
The first tab of VM/VMI is `overview`, then `details`.

Reply:
Made the correction.

Reply:
Looks good.